src/Service.GraphQLBuilder/Directives/RelationshipDirective.cs

```diff
@@ -60,6 +65,20 @@ public static string Target(FieldDefinitionNode field)
             return (string?)arg?.Value.Value;
         }
 
+        /// <summary>
+        /// Gets the target entity name for an input infield with a relationship directive.
+        /// </summary>
+        /// <param name="infield">The input field that is expected to have a relationship directive defined on it.</param>
+        /// <returns>The name of the target entity if the relationship is found, null otherwise.</returns>
+        public static string? GetTargetEntity(IInputField infield)
```

Comment:
If removing usage of `GetTargetEntity`, do we need this function anymore?

Reply:
no we don't

Reply:
removed